### PR TITLE
feat(theme-translations): add extra Korean translation, fix typo

### PR DESCRIPTION
**Pull request by revi**
## Motivation

Updated a few translation string that was left untranslated (probably added after the initial translation?). Those left are because I had no idea where it belongs to. Also fixing a typo with Version.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

This probably does not require test plans.

## Related PRs

N/A

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Hi, thanks for your time on this! #5849 has done some refactorings to the translation system, which will cause merge conflicts in your PR after that one got merged (or vice versa: we merge yours and have to fix the conflicts in that one.) We'd also like to try out the new translation workflow. So while your PR looks good to me (especially the fact that it fixes the "ko only has one plural form" warning), would you kindly stay with us for a while and resolve the conflicts after we merge #5849?

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:27]: "Archive" is just "blog archive", as for this page: https://docusaurus.io/blog/archive

I don't know if just 아카이브 makes sense

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:50]: This is for this page: https://docusaurus.io/tests/docs/tags/a something like "2 docs tagged with "a"". "2 docs tagged" is already translated below so you would have to consider how to splice that with "with "a"" while making it make sense.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:59]: This is the button you see on a mobile when in a doc page:

image

Honestly a naive translation like 메인 메뉴로 돌아가기 would work?

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Not experienced with Korean at all but hope these comments can help you complete the translations

**Comment by revi** [comment]
> Hi, thanks for your time on this! #5849 has done some refactorings to the translation system, which will cause merge conflicts in your PR after that one got merged (or vice versa: we merge yours and have to fix the conflicts in that one.) We'd also like to try out the new translation workflow. So while your PR looks good to me (especially the fact that it fixes the "ko only has one plural form" warning), would you kindly stay with us for a while and resolve the conflicts after we merge #5849?

This one can wait. Just ping me when the 5849 is merged and this can be worked on.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: I don't think so, added as "List of blog articles" based on original translator's work below.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: Gotcha, added.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: It does. Added.

**Review by Josh-Cena — APPROVED** [review]
Thanks 👍

"Blog archive" is not easy to translate. I went for "blog history" in `zh-Hans`, but "blog list" also sounds good

**Comment by slorber** [comment]
Thanks

Theme changes have been merged, it may be simpler to create another new PR with these changes ported to the new system:

https://github.com/facebook/docusaurus/tree/main/packages/docusaurus-theme-translations

**Comment by Josh-Cena** [comment]
At least I wouldn't like seeing a PR closed on my timeline😄 @revi Whether to submit a new one or just resolve conflicts in the current one (or let me do it, if you wish), your call

**Comment by revi** [comment]
@Josh-Cena You can feel free to rebase and merge :-) (Thanks in advance!)

**Comment by Josh-Cena** [comment]
Ah, okay, I'll do it then. Thanks again!

**Comment by Josh-Cena** [comment]
@revi actually, I do not have write access because your fork is an org fork not a personal fork. Do you see a checkbox on the right sidebar to allow write access to maintainers?

**Pull request merged**
